packages/hoppscotch-backend/src/team-collection/input-type.args.ts

```diff
@@ -14,6 +14,13 @@ export class CreateRootTeamCollectionArgs {
 
   @Field({ name: 'title', description: 'Title of the new collection' })
   title: string;
+
+  @Field({
+    name: 'data',
+    description: 'JSON string representing the collection data',
+    nullable: true,
+  })
+  data: string;
```

Comment:
Are we allowing to store the string in the Database?

<img width="358" alt="Screenshot 2023-12-05 at 2 27 26 PM" src="https://github.com/hoppscotch/hoppscotch/assets/30154643/5c62d482-019e-45f9-a296-ad3c94938c14">

```
mutation {
  updateTeamCollection(
    collectionID: "clps2e9ej000bpa5cdculujma"
    newTitle: "team col 2"
    data:""
  ) {
    id
    title
    data
  }
}
```